data-prep/app/model.py

```diff
@@ -12,11 +12,15 @@
 from app.holidaysSA import ourHolidays
 from app.fakeInviteGenerator import createInvites
 
-start_date = date(2016,1,1)
+import json 
+
+start_date = date(2015,1,1)
 end_date = date(2022,7,1)
 current_date = end_date
```

Comment:
Are the dates the range of training data dates?

Reply:
These are the dates for which fake data is generated, training uses all the invites in the db.